### PR TITLE
Accessibility

### PR DESCRIPTION
**Pull request by MoRaad97**
# In this PR we worked on Accessibility:

## Accessibility Report

- Page titles - no issues found.
- Image text alternatives. "check the comments"
- Text headings line - no issues found.
- Color contrast - "check comments"
- Resize - no issue found
- Interaction - no issue found
- Moving content. "check the comments"
- Multimedia - no issues found
- The basic structure of the page - no issue found

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -253,7 +253,7 @@ <h2>Website Protfolio</h2>
       <img
         class="desktop-Illustr"
         src="imgs/illustration-about-me.png"
-        alt=""
+        alt="desktop-illustration"
```
Comment (MoRaad97) [inline index.html:256]: Image text alternatives Fix.

File: css/master.css [hunk 2]
```diff
@@ -149,6 +149,10 @@
   animation-direction: alternate;
 }
 
+.headline .headline-container .text h1:hover {
+  animation: none;
+}
+
```
Comment (MoRaad97) [inline css/master.css:156]: Moving content Fix.

File: css/master.css [hunk 3]
```diff
@@ -268,7 +268,7 @@
 .recent-work .container .projects .main-project .paragraph {
   font-family: "Inter", sans-serif;
   line-height: 24px;
-  color: #091e42;
```
Comment (henrycode460) [inline css/master.css:265]: Text color improved to meet accessibility standard

File: css/master.css [hunk 4]
```diff
@@ -684,6 +684,17 @@
   outline: none;
 }
 
+input::placeholder{
+  color: rgb(30, 29, 29);
```
Comment (henrycode460) [inline css/master.css:688]: font-weight and color change to meet accessibility standard

**Pull request merged**